src/util/url.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * @flow
+ * @file Utility functions for urls
+ * @author Box
+ */
+
+/**
+ * Update a key and value in the URI query parameter string
+ *
+ * @param {string} uri - the uri that contains the potential query parameter string
+ * @param {string} key - the parameter key to be updated/added
+ * @param {mixed} value - the parameter value to be updated/added
+ * @return {string}
+ */
+function updateQueryStringParameter(uri: string, key: string, value: any): string {
```

Comment:
Probably best to not handle query string params yourself, lots of edge cases - Preview uses `jsuri`, see: https://github.com/box/box-content-preview/pull/405

Reply:
dev/peerDep it

Reply:
copied these over from the monolith. will look into `jsuri` which sounds like a better idea